babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -42,4 +42,35 @@ SqlNode DateFunctionCall() :
     }
 }
 
+SqlBinaryOperator SpecialBinaryOperator() :
+{
+}
+{
+    <PG_CAST> { return SqlStdOperatorTable.PG_CAST; }
+}
+
+SqlIdentifier PostgreSQLTypes(Span s) :
+{
+}
+{
+    "REGPROC" { return new SqlIdentifier("REGPROC", s.end(this)); }
+}
```

Comment:
Why we also introduce new `OID` type ?

Reply:
I'll remove that inshallah. Such types can be provided by the schema.